patches/content-browser-service_worker-service_worker_content_settings_proxy_impl.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/content/browser/service_worker/service_worker_content_settings_proxy_impl.h b/content/browser/service_worker/service_worker_content_settings_proxy_impl.h
+index 49a04df206daaf3a1bb7a12b0e79497c26210c39..6c3da673a1edbf9d29965ad40ff5aef5e598fd3e 100644
+--- a/content/browser/service_worker/service_worker_content_settings_proxy_impl.h
++++ b/content/browser/service_worker/service_worker_content_settings_proxy_impl.h
+@@ -41,6 +41,7 @@ class ServiceWorkerContentSettingsProxyImpl final
+   void RequestFileSystemAccessSync(
+       RequestFileSystemAccessSyncCallback callback) override;
+ 
++  BRAVE_SERVICE_WORKER_SETTINGS_PROXY_IMPL_H
```

Comment:
I think you should be able to replace this with `#define` in both `_impl.h` files. Make sure to include mojo header first to not override its internals.

```
#define RequestFileSystemAccessSync                                            \
  AllowFingerprinting(AllowFingerprintingCallback callback) override;          \
  void GetBraveFarblingLevel(GetBraveFarblingLevelCallback callback) override; \
  void RequestFileSystemAccessSync(RequestFileSystemAccessSyncCallback callback)    \
      override;
```

Reply:
fixed in https://github.com/brave/brave-core/commit/eb9411b9f97716dbfb49a6834b3499f3beb36129